package.json

```diff
@@ -132,6 +132,7 @@
     "rollup-plugin-sass": "^1.2.2",
     "rollup-plugin-stylelint": "1.0.0",
     "rollup-plugin-url": "^3.0.1",
+    "sass": "^1.34.0",
```

Comment:
just curious, why do we need to change node-sass with sass package here?

Reply:
Because old node-sass not compile correctly on windows 10 D: (and node-sass depreacted since 26 October 2020)
But I compared, compiled css not changed.

Reply:
Thanks @243083df, lgtm